src/BenchmarkDotNet.Toolchains.Uap/Toolchains/Uap/UapBuilder.cs

```diff
@@ -0,0 +1,62 @@
+﻿#if !UAP
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.Results;
+using System.Diagnostics;
+using System.IO;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+
+namespace BenchmarkDotNet.Toolchains.Uap
+{
+    internal class UapBuilder : IBuilder
+    {
+        private const string Configuration = "Release";
+
+        internal const string OutputDirectory = "binaries";
+
+        private static readonly TimeSpan DefaultTimeout = TimeSpan.FromMinutes(10);
+
+        public BuildResult Build(GenerateResult generateResult, ILogger logger, Benchmark benchmark,
+            IResolver resolver)
+        {
+            if (!ExecuteCommand("cmd /c \"" + generateResult.ArtifactsPaths.BuildScriptFilePath + "\"",
+                generateResult.ArtifactsPaths.BuildArtifactsDirectoryPath,
+                DefaultTimeout))
+            {
+                return BuildResult.Failure(generateResult, new Exception(" failed"));
+            }
+
+            return BuildResult.Success(generateResult);
+        }
+
+        internal static bool ExecuteCommand(string commandWithArguments, string workingDirectory, TimeSpan timeout)
+        {
+            using (var process = new Process { StartInfo = BuildStartInfo(workingDirectory, commandWithArguments) })
+            {
+                process.Start();
+
+                process.WaitForExit((int)timeout.TotalMilliseconds);
```

Comment:
what about the error output of the command when it fails? If we don't print it to the users they will come back to you with every single problem with the new toolchain ;)